src/applications/vaos/services/healthcare-service/transformers.v2.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @module services/HealthcareService/transformers
+ */
+
+/**
+ * Transforms a clinic from the VAOS service into our HealthCareService format
+ *
+ * @param {VAOSClinic} clinic Clinic object to transform to HealthcareService object
+ *
+ * @returns {HealthCareService} A HealthCareService resource
+ */
+export function transformClinicV2(clinic) {
+  return {
+    // ID: VA facility site code and clinic id
+    id: `${clinic.vistaSite}_${clinic.id}`,
```

Comment:
Our format already matches the clinic format, so we could update the id and pass it through, but seemed safer to just pass the properties we know we need.